app/cp_nigeria/migrations/0005_consumergroup_group_id_consumergroup_project_and_more.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.4 on 2023-09-22 18:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0019_timeseries'),
```

Comment:
Before we merge this one we will update the migration file as the latest on  project is 0020 not 0019.